tracker/data/processing.py

```diff
@@ -36,6 +36,51 @@
 SCAN_CACHE = os.path.join(env.SCAN_DATA, "cache")
 SCAN_DOMAINS_CSV = os.path.join(SCAN_CACHE, "domains.csv")
 
+ACCEPTED_CIPHERS = {
```

Comment:
Is there a way to pull these out into a config.py or equivalent?

Reply:
Still relevant if we go dynamic from DB?